bin/docserv

```diff
@@ -65,8 +62,17 @@ class Deliverable:
         self.target = target
         self.product_id = build_instruction['product']
         self.docset_id = build_instruction['docset']
+        self.instruction = build_instruction['id']
         logger.debug("Created deliverable: %s %s %s %s %s" % (local_repo_build_dir, self.language, dc_file, build_format, target['target_path']))
 
+    def id(self):
```

Comment:
Just a minor issue. I think, it would make sense to use a property for that:

```python

@property
def id(self):
   # ....
```

Keep in mind, this would be not a function anymore. You would use it as `obj.id` instead of `obj.id()`.